README.md

```diff
@@ -14,3 +14,59 @@
 1. Ground Textures: https://opengameart.org/content/2d-minecraft-sprite-pack
 2. Steve Textures: https://opengameart.org/content/2d-minecraft-steve
 3. Zombie Sprite: https://opengameart.org/content/zombie-character-sprite
+
+Minecraft
+<div>
+  <img src="images/minecraft/Spritesheet.PNG?raw=true" width=50 height=50 hspace=3>
```

Comment:
Let's not show spritesheet as a 50x50. Let's show them in full size.

Reply:
I fixed this for the spritesheets in the "general" section. I removed the one in "Minecraft" as it just showed the same assets that comes right after.